src/routes/(protected)/calendar/+page.svelte

```
@@ -322,54 +322,21 @@
+                        <Checkbox
+                            id={opt.id}
+                            checked={filters[opt.type]}
+                            onCheckedChange={(checked) => (filters[opt.type] = !!checked)} />
+                        <label
+                            for={opt.id}
+                            class="flex cursor-pointer items-center gap-2 text-sm font-medium">
+                            <Icon class={cn("h-4 w-4", typeStyles[opt.type].icon)} />
+                            {opt.label}
+                        </label>
+                    </div>
+                {/each}
```

ESLint

[error] Each block should have a key

(svelte/require-each-key)